esmvaltool/preprocessor/_area.py

```diff
@@ -79,7 +79,39 @@ def extract_region(cube, start_longitude, end_longitude, start_latitude,
     return cube
 
 
-# get zonal means
+def get_iris_analysis_operation(operator):
+    """
+    Determine the iris analysis operator from a string.
+
+    Arguments
+    ---------
+        operator: string
+            A named operator.
+    Returns
+    -------
+        function: A function from iris.analysis
+    """
+    operators = ['mean', 'median', 'stdev', 'variance', 'min', 'minimum',
```

Comment:
Why are there two ways to specify min and max, and why does our standard deviation have a different name than in iris? I think one way is good enough and using the same names as iris is more clear.

Also, I think it would be cleaner to implement this function as:
```python
operators = ['mean', 'median', 'std_dev', 'variance', 'min', 'max']
if operator not in operators:
    raise ValueError(
        f"operator '{operator}' not recognised. Accepted values are: {operators}")
operation = getattr(iris.analysis, operator.upper())
```